whatsup/asgi.py

```
@@ -1,13 +1,24 @@
 import os
 from django.core.asgi import get_asgi_application
-from channels.routing import ProtocolTypeRouter, URLRouter
-import chats.routing
+from dotenv import load_dotenv
```

Ruff (0.8.2)

`dotenv.load_dotenv` imported but unused

Remove unused import: `dotenv.load_dotenv`

(F401)